Tools/ardupilotwaf/boards.py

```diff
@@ -708,177 +708,3 @@ def configure_env(self, cfg, env):
-        env.PX4_PX4IO_NAME = self.px4io_name
-        env.PX4_PARAM_DEFAULTS = self.param_defaults
-        env.PX4_RC_S_SCRIPT = 'init.d/rcS'
-
-        env.AP_PROGRAM_AS_STLIB = True
-
-    def build(self, bld):
-        super(px4, self).build(bld)
-        bld.ap_version_append_str('NUTTX_GIT_VERSION', bld.git_submodule_head_hash('PX4NuttX', short=True))
-        bld.ap_version_append_str('PX4_GIT_VERSION', bld.git_submodule_head_hash('PX4Firmware', short=True))
-        bld.load('px4')
-
-    def romfs_exclude(self, exclude):
-        self.ROMFS_EXCLUDE += exclude
-
-class px4_v1(px4):
-    name = 'px4-v1'
-    def __init__(self):
-        super(px4_v1, self).__init__()
-        self.bootloader_name = 'px4fmu_bl.bin'
-        self.board_name = 'px4fmu-v1'
-        self.px4io_name = 'px4io-v1'
-        self.romfs_exclude(['oreoled.bin'])
-
-class px4_v2(px4):
-    name = 'px4-v2'
-    def __init__(self):
-        super(px4_v2, self).__init__()
-        self.bootloader_name = 'px4fmuv2_bl.bin'
-        self.board_name = 'px4fmu-v2'
-        self.px4io_name = 'px4io-v2'
-        self.romfs_exclude(['oreoled.bin'])
-        self.with_uavcan = True
-
-class px4_v3(px4):
-    name = 'px4-v3'
-    def __init__(self):
-        super(px4_v3, self).__init__()
-        self.bootloader_name = 'px4fmuv2_bl.bin'
-        self.board_name = 'px4fmu-v3'
-        self.px4io_name = 'px4io-v2'
-        self.with_uavcan = True
-
-class skyviper_v2450_px4(px4_v3):
-    name = 'skyviper-v2450-px4'
-    def __init__(self):
-        super(skyviper_v2450_px4, self).__init__()
-        self.px4io_name = None
-        self.param_defaults = '../../../Tools/Frame_params/SkyViper-2450GPS/defaults.parm'
-
-    def configure_env(self, cfg, env):
-        super(skyviper_v2450_px4, self).configure_env(cfg, env)
-
-        env.DEFINES.update(
-            TOY_MODE_ENABLED = 'ENABLED',
-            USE_FLASH_STORAGE = 1,
-            ARMING_DELAY_SEC = 0,
-            LAND_START_ALT = 700,
-            HAL_RCINPUT_WITH_AP_RADIO = 1,
-            LAND_DETECTOR_ACCEL_MAX = 2,
-            CYRF_SPI_PX4_SPI_BUS = 2,
-            CYRF_SPI_PX4_SPIDEV_EXT = '(spi_dev_e)1',
-            CYRF_IRQ_INPUT = '(GPIO_INPUT|GPIO_FLOAT|GPIO_EXTI|GPIO_PORTD|GPIO_PIN15)',
-        )
-        env.PX4_RC_S_SCRIPT = 'init.d/rcS_no_microSD'
-        env.BUILD_ABIN = True
-
-class px4_v4(px4):
-    name = 'px4-v4'
-    def __init__(self):
-        super(px4_v4, self).__init__()
-        self.bootloader_name = 'px4fmuv4_bl.bin'
-        self.board_name = 'px4fmu-v4'
-        self.romfs_exclude(['oreoled.bin'])
-        self.with_uavcan = True
-
-class px4_v4pro(px4):
-    name = 'px4-v4pro'
-    def __init__(self):
-        super(px4_v4pro, self).__init__()
-        self.bootloader_name = 'px4fmuv4pro_bl.bin'
-        self.board_name = 'px4fmu-v4pro'
-        self.px4io_name = 'px4io-v2'
-        self.romfs_exclude(['oreoled.bin'])
-        self.with_uavcan = True		
-
-class aerofc_v1(px4):
```

Comment:
Yes, I'm aware. I have a partial port to chibios, but no ETA when that will be completed. I can live with master without it and catch up later since I don't want  to be a blocker in ehat i see is the right direftion. I'd appreciate a little more time until we do a release though